Dockerfile.proto

```diff
@@ -14,19 +13,20 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /tmp
+ARG PROTOBUF_VERSION=21.0
 RUN wget -O protobuf-cpp-"${PROTOBUF_VERSION}".tar.gz \
-    https://github.com/protocolbuffers/protobuf/releases/download/v"${PROTOBUF_VERSION}"/protobuf-cpp-"${PROTOBUF_VERSION}".tar.gz \
+    https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-cpp-3.${PROTOBUF_VERSION}.tar.gz \
```

Comment:
```suggestion
    https://github.com/protocolbuffers/protobuf/releases/download/v3.${PROTOBUF_VERSION}/protobuf-cpp-3.${PROTOBUF_VERSION}.tar.gz \
```
not sure..

Reply:
no, this was correct. They changed the versioning scheme from 3.X to just X last year, to distinguish between "major" changes to language-specific bindings vs the version of the protocol schema itself (version 3). So the GitHub tag is `v21.0` but the source archive is `cpp-3.21.0`